src/executables/util_cs.c

```diff
@@ -4649,7 +4651,69 @@ memmon (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
+  /* execute phase */
+  error_code = mmon_get_server_info (server_info);
+  if (error_code != NO_ERROR)
+    {
+      goto error_exit;
+    }
+
+  // TODO: mmon_print_server_info (server_info);
+
+  if (module_index >= 0)
+    {
+      error_code = mmon_get_module_info (module_index, module_info);
+      if (error_code != NO_ERROR)
+	{
+	  goto error_exit;
+	}
+
+      // TODO: mmon_print_module_info (module_info);
+    }
+
+  if (!transaction && !module)
+    {
+      /* default case */
+      // *INDENT-OFF*
+      error_code = mmon_get_module_info_summary (std::min ((int) MMON_MODULE_LAST, 5), module_info);
+      // *INDENT-ON*
+      if (error_code != NO_ERROR)
+	{
+	  goto error_exit;
+	}
+
+      // TODO: mmon_print_module_info_summary (module_info);
+
+      transaction = true;
```

Comment:
```suggestion
      does_print_transaction = true;
```
or whatever to distinguish a flag variable.